### PR TITLE
Parametrize latency pod cpu and memory requests in load test

### PR DESCRIPTION
**Pull request by marseel**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Allows to change requests for latency pods in load test
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: clusterloader2/testing/load/config.yaml [hunk 1]
```diff
@@ -21,6 +21,13 @@
 {{$ENABLE_NETWORKPOLICIES := DefaultParam .CL2_ENABLE_NETWORKPOLICIES false}}
 {{$ENABLE_DNSTESTS := DefaultParam .CL2_ENABLE_DNSTESTS false}}
 {{$ENABLE_DENSITY_TEST := DefaultParam .CL2_ENABLE_DENSITY_TEST false}}
+# LATENCY_POD_MEMORY and LATENCY_POD_CPU are calculated for 1-core 4GB node.
+# Increasing allocation of both memory and cpu by 10%
+# decreases the value of priority function in scheduler by one point.
+# This results in decreased probability of choosing the same node again.
+# TODO(https://github.com/kubernetes/perf-tests/issues/1024): See whether we can get rid of this
+{{$LATENCY_POD_CPU := DefaultParam .LATENCY_POD_CPU 100}}
```
Comment (mborsz) [inline clusterloader2/testing/load/config.yaml:29]: Is it intentional that you don't use CL2_ prefix?

Comment (marseel) [reply clusterloader2/testing/load/config.yaml:29]: hm, I took these parameter names from density/config.yaml

Comment (marseel) [reply clusterloader2/testing/load/config.yaml:29]: I've added CL2_ prefix

**Pull request merged**